### PR TITLE
Final Review

### PR DESCRIPTION
**Pull request by zonesoldier**
This is the final review of the project.

Let me know if everything works.

You can visit my built at zonesoldier.com

### CODE REVIEW

File: js/card.js [hunk 1]
```diff
@@ -570,7 +575,37 @@ cardOBJ = [
 ,[548,100,0,0,52,5,51,5,312,20,311,20,0]
 ,[549,100,0,0,52,5,51,5,302,30,301,30,0]
 ,[550,100,0,"Increase the probability of causing Sleep by an additional 2%.",155,10,60,5,0]
+,[551,3,"Bangungot",0,77,20,79,15,151,30,153,30,154,30,159,30,0]
+,[552,3,"Bakonawa","For Non-Monk classes, increase physical damage against Boss monsters by 15%.<br/>For Monk classes, increase physical damage against Boss monsters by 10%.",77,20,79,-35,0]
+,[553,3,"Buwaya",0,77,20,79,-35,97,7,0]
+,[554,2,"Bungisngis","For every refine after +5, MaxHP + 1%, MaxSP + 1%.",15,5,16,5,0]
+,[555,2,"Engkanto","Ignore 25% of Plant monster defense.",0,45,25,0]
```
Comment (mickael-bouvart-mpf) [inline js/card.js]: Please remove the "0," after the custom description, so it becomes
`,[555,2,"Engkanto","Ignore 25% of Plant monster defense.",45,25,0]`

Comment (Hatfun) [inline js/card.js]: Please remove the "0," so it becomes
`,[555,2,"Engkanto","Ignore 25% of Plant monster defense.",45,25,0]`

File: js/card.js [hunk 2]
```diff
@@ -1136,6 +1171,17 @@ cardID = [
...
+,[552,1,0]
+,[553,1,0]
+,[554,1,0]
+,[555,1,0]
+,[556,1,0]
+,[557,1,0]
+,[558,1,0]
+,[559,1,0]
+,[560,1,0]
+,[561,1,0]
```
Comment (Hatfun) [inline js/card.js]: Monster IDs:

- Bangungot: 4590
- Bakonawa: 4591
- Buwaya: 4592
- Bungisngis: 4582
- Engkanto: 4583
- Manananggal: 4584
- Mangkukulam: 4585
- Tikbalang: 4586
- Tiyanak: 4587
- Wakwak: 4588
- Jejeling: 4589

File: js/card.js (hunk 1 above)
Comment (zonesoldier) [reply js/card.js]: Changed, thank you.

File: js/card.js (hunk 2 above)
Comment (zonesoldier) [reply js/card.js]: Excellent, i made the changes.

**Comment by zonesoldier** [comment]
Made the changes suggested. Closing.

**Pull request closed without merging**